README.md

```diff
@@ -54,6 +56,17 @@ In addition to standards provided by [default](https://github.com/squizlabs/PHP_
 * WordPress-VIP
 * Zend
 
+### Custom Standards
+
+You can also include a custom phpcodesniffer standard configuration file. Add
+the file to the engine config standard section as a relative path:
+
+    engines:
+      phpcodesniffer:
+        enabled: true
+        config:
+          standard: "phpcs.ruleset.xml"
```

Comment:
Was it intentional to make this a first-class section? I think it'd be simpler to do what we do for PHPMD, by including a relative-ruleset in the existing example configuration.

```yaml
engines:
  phpmd:
    enabled: true
    config:
      file_extensions: "php"
      rulesets: "unusedcode,codesize,naming,optional_relative_path_to_custom_ruleset.xml"
```